### PR TITLE
Remove inline HTML

### LINKED ISSUE #102: Remove inline HTML
The reason we want to use hugo is not to use any HTML markdown in the content docs.

* Remove all {{% sc_ulist %}}
* Remove all &nbsp;

### PR DESCRIPTION
**Pull request by rakyll**
Fixes #102.

### CODE REVIEW

**Comment by rakyll** [comment]
Depends on https://github.com/census-instrumentation/opencensus-website/pull/124.

**Comment by songy23** [comment]
Do you want to rebase this branch?

**Comment by rakyll** [comment]
On it!

**Comment by rakyll** [comment]
PTAL, rebased.

File: content/glossary.md [hunk 1]
```diff
@@ -105,47 +87,35 @@ Tags are key-value pairs that are used to store information about metrics for ex
...
 
 ---
-&nbsp;  
 
-**RPC**  
-Remote Procedure Call. The mechanism of invoking a procedure/subroutine in a different scope/address space.  
-&nbsp;  
+**Remote Procedure Call**
+
+The mechanism of invoking a procedure/subroutine in a different scope/address space.
```
[the remark this answers is not in the document]
Comment (rakyll) [reply content/glossary.md]: Done, replaced the title with RPC.

File: content/go.md [hunk 2]
```diff
@@ -10,45 +10,40 @@ date = "2018-05-17T14:17:26-05:00"
...
 #### Example
 This guide helps you write a Go program instrumented with OpenCensus.  
-&nbsp;
 
 **Prerequisites**  
 
 Go 1.8 or higher is required. Make sure you have 1.8+ installed by running:
 
-``` go
+```
```
[the remark this answers is not in the document]
Comment (rakyll) [reply content/go.md]: Done.

File: content/java.md [hunk 3]
```diff
@@ -9,58 +9,48 @@ date = "2018-05-18T09:59:40-05:00"
...
 
 The OpenCensus Java artifacts are released to Maven Central [{{< sc_gloss1 >}}maven.org{{< /sc_gloss1 >}}](http://search.maven.org/), each contains the jar file, the associated javadoc, and the associated source code. The OpenCensus Java API artifact (along with the associated javadoc and source) is available on Maven Central here: [{{< sc_gloss1 >}}Open Census API{{< /sc_gloss1 >}}](https://search.maven.org/#search%7Cga%7C1%7Copencensus%20api)  
-&nbsp;  
 
 ---
 
-&nbsp;  
 #### Example
-1. Clone the OpenCensus Java github repository:
+Clone the OpenCensus Java GitHub repository:
```
[the remark this answers is not in the document]
Comment (rakyll) [reply content/java.md]: Done.

File: content/php.md [hunk 4]
```diff
@@ -9,16 +9,13 @@ date = "2018-05-18T12:37:28-05:00"
...
-&nbsp;
+
 #### API Documentation
 The OpenCensus libraries artifacts are released to Packagist ([{{< sc_gloss1 >}}packagist.org{{< /sc_gloss1 >}}](https://packagist.org/)) [{{< sc_gloss1 >}}opencensus/opencensus{{< /sc_gloss1 >}}](https://packagist.org/packages/opencensus/opencensus). The API documentation is available [{{< sc_gloss1 >}}here{{< /sc_gloss1 >}}](https://census-instrumentation.github.io/opencensus-php/api/).  
-&nbsp;  
 
 ---
-&nbsp;
 #### Example
 1. Clone the OpenCensus PHP github repository:
```
[the remark this answers is not in the document]
Comment (rakyll) [reply content/php.md]: Added them back to Java instead.

**Pull request merged**